ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -13,15 +13,15 @@ Imagine being asked to configure a **layer 3 leaf spine (L3LS)** network from sc
 ![Figure: Arista Leaf Spine topology](../_media/getting-started/single-dc-topology-physical.png)
 
 Traditionally one would configure the switches manually using a laptop, console cable, and USB key to load the correct EOS software
-image and configuration. Of course, the configuration would be manually generated, using snippets from the relevant Arista design guides, or copying/pasting from existing devices. Configurations would have to be manually adapted to each and every device, which is time-consuming and prone to errors. Consider what would happen if a new pair of leaf switches or new VRFs/VLANs are added to the network - changes would need to be made to most of the devices.
+image and configuration. Of course, the configuration would be manually generated, using snippets from the relevant Arista design guides, or copying/pasting from existing devices. Configurations would have to be manually adapted to each and every device, which is time-consuming and prone to errors. Consider what would happen if a new pair of leaf switches or new VRFs/VLANs were added to the network - changes would need to be made to most of the devices.
```

Comment:
I'd simplify:
Imagine a new pair of leaf switches or new VRFs/VLANs were added to the network - changes would need to be made to most of the devices.

Reply:
addressed in 6e84056